batch_job/src/vine_factory.c

```diff
@@ -219,6 +219,34 @@ int manager_workers_capacity(struct jx *j) {
 	return capacity;
 }
 
+char *shell_escape(const char *input) {
```

Comment:
Could we make it work with `string_escape_shell` in `dttools/src/stringtools.c`?